proposals/index.md

```diff
@@ -0,0 +1,5 @@
+# Table of Contents
+
+## Proposals
```

Comment:
should this header and folder be called "proposals" given that this is the final outcome from the discussions related to the proposal ? one suggestion is to use docs/design/foobar.md

Reply:
Moved things around a bit. I didn't realize we had a docs folder. I'll check if I have to move that content to the public site.